packages/design-system/src/components/utilities/typography.stories.tsx

```diff
@@ -44,12 +45,18 @@ export const Body = () => {
 
 // an example for responsive headings -- referenced in doc site
 export const ResponsiveHeadings = () => (
-  <article className="ds-base--inverse ds-u-padding--2">
-    <h1 className="ds-text-heading--5xl ds-u-margin-y--1">.ds-text-heading--5xl</h1>
-    <h2 className="ds-text-heading--4xl ds-u-margin-y--1">.ds-text-heading--4xl</h2>
-    <h3 className="ds-text-heading--3xl ds-u-margin-y--1">.ds-text-heading--3xl</h3>
-    <h4 className="ds-text-heading--2xl ds-u-margin-y--1 ds-u-font-size--h4 ds-u-md-font-size--h3 ds-u-lg-font-size--h2">
-      Responsive utility classes
-    </h4>
-  </article>
+  <>
+    <h1 className="ds-text-heading--5xl">We the People of the United States (5xl)</h1>
+    <h2 className="ds-text-heading--4xl">We the People of the United States (4xl)</h2>
+    <h3 className="ds-text-heading--3xl">We the People of the United States (3xl)</h3>
```

Comment:
I've never noticed this until now, but 3xl and lower have top-margin but 5xl and 4xl do not. Makes this story look kind of funny

<img width="877" alt="Screen Shot 2022-10-18 at 2 12 57 PM" src="https://user-images.githubusercontent.com/7595652/196546350-9eb17a09-86f6-49b6-85e4-e505b4037c3b.png">

Reply:
I agree, I was thinking about breaking this story out into 3 separate stories or possibly incorporate them into the heading at the top of the page

Reply:
That's a good question, and I don't know 🤷‍♂️

Reply:
The spacing isn't really what the story is about.

Reply:
I don't think we should add any utility classes to any of our examples b/c that would be confusing for someone consuming. How would they know when to use an additional utility and when not to also it makes our example inaccurate.   
  
I have also noticed that for the other heading examples above on the this page that we have some doc site styles that are impacting the spacing that should be showing up in the example. I think this gives a false sense of what is really going on in the example.   
  
Here is a snippet of CSS that impacts the headers `:last-child` and `:first-child` overrides the default spacing for a few of these examples :-/   
 

![](https://user-images.githubusercontent.com/1449852/197207115-ea6e6b50-0ee9-42b2-be04-fb9d06c6c1c2.png)